src/Jobs/DataTableExportJob.php

```diff
@@ -179,6 +180,10 @@ public function handle()
         }
 
         $writer->close();
+
+        if($this->getS3Disk()) {
+            Storage::disk($this->getS3Disk())->put($filename, (new File($path))->getContent());
```

Comment:
Wouldn't this exceed memory since you are reading the whole contents of the file?